src/ScottLogic.Internal.Training/ScottLogic.Internal.Training.Matcher/Trade.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace ScottLogic.Internal.Training.Matcher
+{
+    public class Trade : IEquatable<Trade>
+    {
+        private int AccountNumber;
+        public int Quantity { get; set; }
```

Comment:
Personal preference but I would usually put private fields before properties.

The way I tend to organise my classes are:

fields

properties

contructors

methods.